common/lib/capa/capa/tests/test_files/targeted_feedback.xml

```diff
@@ -2,7 +2,7 @@
     <p>What is the correct answer?</p>
     <multiplechoiceresponse targeted-feedback="">
       <choicegroup type="MultipleChoice">
-        <choice correct="false" explanation-id="feedback1">wrong-1</choice>
+        <choice correct="false" explanation-id="feedback1" role="group" aria-live="polite" aria-describedby="1_2_1-legend">wrong-1</choice>
```

Comment:
This does not look correct. These elements should be on the targetedfeedback (as in test_targeted_feedback), not on the choice.